### ISSUE TITLE
Profiling window crashes BasinC

### ISSUE THREAD

**Issue by lippmaje** [ref-xx/basinc#19]
v1.794 on Win 7 SP1

The profiling window sometimes fails to get a handle and totally messes up its window content.
This causes BasinC to lose control over all of its other windows, and trying to close BasinC results in a crash while all the windows contents remain on the screen.
The only way to end BasinC now is to shut it down by the Windows manager. Your program may get lost, depending on if you managed to click the SAVE button successfully which often fails because the 'Canvas does not allow drawing.' alert box keeps popping up permanently.
Screens below.

Steps to recreate:
 - enable profiling and run a program
 - open the profile viewer
 - scroll a bit around, close and reopen the window, bring some other window to front and then refocus the profile viewer, etc (in some random order)

The bug happened also in earlier versions, I'm just reporting it now.
 
![screen](https://github.com/ref-xx/basinc/assets/87007255/31ac2deb-8e1d-4771-8ecf-d43066c5bcde)
![screen2](https://github.com/ref-xx/basinc/assets/87007255/d03bd052-5629-4714-850d-e9ce9f2fcaec)
![screen3](https://github.com/ref-xx/basinc/assets/87007255/29b9182c-90f1-4043-a0d8-94506c1d4d04)
![screen4](https://github.com/ref-xx/basinc/assets/87007255/d81c7bc2-8e63-4814-9fd6-dc0bb07cee25)

**Comment by ref-xx** [comment]
I cannot replicate the bug here, but it looks like the drawing object loses the handle to draw. Added to the end of the looong bug list :) 

btw, do you use any antivirus or process monitoring program?

thanks.

**Comment by lippmaje** [comment]
No, just Win 7 SP1 (classic theme). Here's a test program and another screen.
![screen5](https://github.com/ref-xx/basinc/assets/87007255/239c2e26-6024-4580-bb56-763e63b7f620)
[profilebug.zip](https://github.com/ref-xx/basinc/files/14323767/profilebug.zip)

To trigger the bug I scrolled in the results, selected a line here and there, switched between editor and this window, closed/reopened/resized the window in some arbitrary order. It takes a few tries until it happens.

**Comment by ref-xx** [comment]
it looks like it's an old and known Canvas bug. 

https://stackoverflow.com/questions/17087929/setting-brush-color-in-listviews-oncustomdraw-functions-crashes-the-application

I'm looking for workarounds, it stops crashing if I don't change the color of blue stripes, and listview basically stack overflows because of open handles.  Thank you for reporting.

**Comment by ref-xx** [comment]
@lippmaje can you check 1.8 version to see if it still crashes on your computer?

**Comment by ref-xx** [comment]
Ok, I'm closing this as this looks like windows problem, which is gone windows10+.
